charts/rancher-monitoring/v0.2.3/charts/operator-init/files/crd-alertmanager.yaml

```diff
@@ -1,9 +1,8 @@
-# https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/d8d7949edf839ffb7fae859992c05c894b373862/bundle.yaml
+# https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.40.0/bundle.yaml
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    helm.sh/hook: crd-install
```

Comment:
why is this removed? 
Is it removed by the upstream?

Reply:
yes it is removed by upstream and yes we effectively do the CRD (sub)chart install in Monitoring V1